pages/exercises/[lessonSlug].tsx

```diff
@@ -78,12 +64,21 @@ const Exercises: React.FC<QueryDataProps<GetExercisesQuery>> = ({
           lessonTitle={currentLesson.title}
           hasPrevious={exerciseIndex > 0}
           hasNext={exerciseIndex < currentExercises.length - 1}
+          submitUserAnswer={(userAnswer: string) =>
+            setUserAnswers({ ...userAnswers, [exercise.id]: userAnswer })
```

Comment:
Here we add the user's answer to our userAnswers object which stores all the user's answers by exercise ID.